core/tests/protractor_utils/workflow.js

```diff
@@ -58,19 +58,13 @@ var createAndPublishExploration = function(
     title, category, objective, language) {
   createExploration();
   editor.setContent(forms.toRichText('new exploration'));
-  editor.setInteraction('TextInput');
-  editor.setDefaultOutcome(null, 'final state', true);
+  editor.setInteraction('EndExploration');
```

Comment:
Interesting -- I get the idea of making this simpler; that's a good idea and it saves time. Just to check, though, do they break any of the existing tests?